ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py

```diff
@@ -50,23 +48,29 @@ def test_tensorflow_shuffle(
         available_dtypes=ivy_tf.valid_float_dtypes,
         min_value=-1000,
         max_value=100,
+        allow_inf=False,
     ),
     dtype_and_high=helpers.dtype_and_values(
         available_dtypes=ivy_tf.valid_float_dtypes,
         min_value=101,
         max_value=1000,
+        allow_inf=False,
     ),
-    as_variable=st.booleans(),
+    shape=helpers.get_shape(),
     num_positional_args=helpers.num_positional_args(
         fn_name="ivy.functional.frontends.tensorflow.uniform"
     ),
-    native_array=st.booleans(),
 )
 def test_tensorflow_uniform(
-    dtype_and_low, dtype_and_high, as_variable, num_positional_args, native_array, fw
+    dtype_and_low, dtype_and_high, shape, as_variable, num_positional_args, native_array, fw
 ):
     low_dtype, low = dtype_and_low
     high_dtype, high = dtype_and_high
+    # If low and high are numerics and not arrays, shape=shape else shape=None
+    if isinstance(low, (int, float)) and isinstance(high, (int, float)):
+        shape = shape
+    else:
+        shape = None
     helpers.test_frontend_function(
```

Comment:
Hello @Auxia, can you make sure to also generate a value for the dtype argument and pass it to the test as well ? Keep in mind that it also affects the default maximum value of the function in case of floating dtypes as mentioned in the docs [here](https://www.tensorflow.org/api_docs/python/tf/random/uniform#args)